Marksman/Refs.fs

```diff
@@ -316,17 +322,27 @@ module Dest =
                 | Dest.LinkDef _ -> [ srcDoc ]
                 | Dest.Heading _
                 | Dest.Doc _ -> Folder.docs folder |> List.ofSeq
+                | Dest.Tag _ -> Folder.docs folder |> List.ofSeq
 
             let referencingEls =
                 seq {
                     for targetDoc in targetDocs do
-                        let targetDocRefs = resolveLinks folder targetDoc
-
-                        let backRefs =
-                            findReferencingElements targetDocRefs declToFind
-                            |> Seq.map (fun (el, dest) -> targetDoc, el, dest)
-
-                        yield! backRefs
+                        match declToFind with
+                        | Dest.Tag(_, tag) ->
+                            let targets =
+                                Doc.index targetDoc
+                                |> Index.filterTagsByName tag.text
```

Comment:
You need
```suggestion
                                |> Index.filterTagsByName tag.data.name.text
```

tag.text gives you '#tagname', while 'tag.data.name.text' gives 'tagname' without the leading '#'